javascript/javascript3/week1/Giphy_Api/giphy.js

```diff
@@ -0,0 +1,28 @@
+// Resting the page
+
+function sampleFunction() {
+    location.reload();
+}
```

Comment:
I think it is not a good idea to reload the page at [SPA](https://en.wikipedia.org/wiki/Single-page_application#:~:text=From%20Wikipedia%2C%20the%20free%20encyclopedia,browser%20loading%20entire%20new%20pages.)

You could create a `div` element, for example
```html
<div id="images"></div>
```
Then append created images into it instead of body
```js
images.appendChild(...)
```
Then to reset or remove them instead of reloading the page you can remove them something like this
```js
images.querySelectorAll('img').forEach(img => img.remove())
```